addons/payment/tests/test_flows.py

```diff
@@ -347,3 +327,41 @@ def test_transaction_wrong_token(self):
         self.assertIn(
             "odoo.exceptions.ValidationError: The access token is invalid.",
             response.text)
+
+    @mute_logger('odoo.addons.payment.models.payment_transaction')
+    def _test_max_one_payment_request_from_portal(self, flow):
+
+        payment_option_id = self.create_token().id if flow == 'token' else self.acquirer.id
+        data = {
+            'amount': self.amount,
+            'currency_id': self.currency.id,
+            'partner_id': self.partner.id,
+            'access_token': None, flow: flow,
+            'payment_option_id': payment_option_id,
+            'reference_prefix': 'test',
+            'tokenization_requested': True,
+            'landing_route': 'Test',
+            'is_validation': False,
+            'invoice_id': self.invoice.id,
+            'flow': flow,
+        }
+
+        with unittest.mock.patch(
+                'odoo.addons.payment.utils.check_access_token', return_value=True
+        ), \
+                unittest.mock.patch(
+                    'odoo.addons.payment.models.payment_transaction.PaymentTransaction.'
+                    '_send_payment_request'
+                ) as patched:
+
+            self.portal_transaction(**data)
```

Comment:
Please check the request response, to make sure the `assert_not_called` doesn't succeed because the route is broken.